client/package.json

```diff
@@ -94,7 +94,7 @@
 		"@wordpress/components": "^28.2.0",
 		"@wordpress/compose": "^7.2.0",
 		"@wordpress/data": "^10.2.0",
-		"@wordpress/dataviews": "patch:@wordpress/dataviews@npm%3A0.4.1#~/.yarn/patches/@wordpress-dataviews-npm-0.4.1-2c01fa0792.patch",
```

Comment:
Oh, I just realized this hasn't been updated to 4.2.0. Need to make sure this is also updated.

Reply:
Updated here: https://github.com/Automattic/wp-calypso/pull/93503/commits/74ddd8abca8db52231e84134d2e90689eb891a4b